usr/share/rear/lib/bootloader-functions.sh

```diff
@@ -512,20 +518,25 @@ insmod ext2
 set timeout=5
 
 search --no-floppy --file /boot/efiboot.img --set
-#set root=(cd0)
+$grub2_set_usb_root
```

Comment:
Please add a counterpart comment here about how grub2_set_usb_root is set
and what its usual value is so that the code here is easier to understand like
```
# In output/USB/Linux-i386/100_create_efiboot.sh we set
# grub2_set_usb_root="search --no-floppy --set=root --label ${EFI_LABEL}"
# with EFI_LABEL="REAR-EFI":
$grub2_set_usb_root
```

Reason:
I got stopped here while reviewing wondering about
what that "lonely" `$grub2_set_usb_root` is about ;-)

Reply:
it is commented but elsewhere ;-)
https://github.com/rear/rear/pull/2326/files#diff-78c41a8ef5963b8d7fe6c7e84d437046R85

Since this code creates verbatim configuration, I did now want to spoil it with comments.

Reply:
But if you think it is a good idea, I can add it, no problem with that.

Reply:
Strange, I thought I had added a comment here
but now I see there is none...

No need to add comments to code that creates verbatim configuration.
I had not noticed that this is the case here - I had overlooked that this
code is between `cat << EOF` and `EOF`.